<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScaleAndBiasDlg</class>
 <widget class="QDialog" name="ScaleAndBiasDlg">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>518</width>
    <height>256</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::DefaultContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Mesh Scale and Bias</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_meshDesc">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Mesh Description</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Info</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_currentDim">
        <property name="text">
         <string>Before scale and bias: Size: [1, 1, 1] Center: [0, 0, 0]</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_targetDim">
        <property name="text">
         <string>After scale and bias: Size: [1, 1, 1] Center: [0, 0, 0]</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_Scale">
     <property name="title">
      <string>Scale</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QDoubleSpinBox" name="doubleSpinBox_Sx">
        <property name="decimals">
         <number>8</number>
        </property>
        <property name="minimum">
         <double>-999999999.000000000000000</double>
        </property>
        <property name="maximum">
         <double>999999999.000000000000000</double>
        </property>
        <property name="value">
         <double>1.000000000000000</double>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_7">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>x</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="doubleSpinBox_Sy">
        <property name="decimals">
         <number>8</number>
        </property>
        <property name="minimum">
         <double>-999999999.000000000000000</double>
        </property>
        <property name="maximum">
         <double>999999999.000000000000000</double>
        </property>
        <property name="value">
         <double>1.000000000000000</double>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>y</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="doubleSpinBox_Sz">
        <property name="decimals">
         <number>8</number>
        </property>
        <property name="minimum">
         <double>-999999999.000000000000000</double>
        </property>
        <property name="maximum">
         <double>999999999.000000000000000</double>
        </property>
        <property name="value">
         <double>1.000000000000000</double>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_8">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>z</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Bias</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QDoubleSpinBox" name="doubleSpinBox_Bx">
        <property name="decimals">
         <number>8</number>
        </property>
        <property name="minimum">
         <double>-999999999.000000000000000</double>
        </property>
        <property name="maximum">
         <double>999999999.000000000000000</double>
        </property>
        <property name="value">
         <double>0.000000000000000</double>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>x</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="doubleSpinBox_By">
        <property name="decimals">
         <number>8</number>
        </property>
        <property name="minimum">
         <double>-999999999.000000000000000</double>
        </property>
        <property name="maximum">
         <double>999999999.000000000000000</double>
        </property>
        <property name="value">
         <double>0.000000000000000</double>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>y</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="doubleSpinBox_Bz">
        <property name="decimals">
         <number>8</number>
        </property>
        <property name="minimum">
         <double>-999999999.000000000000000</double>
        </property>
        <property name="maximum">
         <double>999999999.000000000000000</double>
        </property>
        <property name="value">
         <double>0.000000000000000</double>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>z</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QFrame" name="frame_2">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="pushButton_ScalToFit1">
           <property name="text">
            <string>Fit to unit cube (isotropic)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_ScalToFit2">
           <property name="text">
            <string>Fit to unit cube (unisotropic)</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QDialogButtonBox" name="buttonBox">
        <property name="locale">
         <locale language="English" country="UnitedStates"/>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="standardButtons">
         <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ScaleAndBiasDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>507</x>
     <y>244</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>255</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ScaleAndBiasDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>507</x>
     <y>244</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>255</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_ScalToFit1</sender>
   <signal>clicked()</signal>
   <receiver>ScaleAndBiasDlg</receiver>
   <slot>ScaleIsotropic()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>65</x>
     <y>231</y>
    </hint>
    <hint type="destinationlabel">
     <x>4</x>
     <y>137</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_ScalToFit2</sender>
   <signal>clicked()</signal>
   <receiver>ScaleAndBiasDlg</receiver>
   <slot>ScaleUnisotropic()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>187</x>
     <y>222</y>
    </hint>
    <hint type="destinationlabel">
     <x>2</x>
     <y>164</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>doubleSpinBox_Sx</sender>
   <signal>valueChanged(QString)</signal>
   <receiver>ScaleAndBiasDlg</receiver>
   <slot>ValuesChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>108</x>
     <y>132</y>
    </hint>
    <hint type="destinationlabel">
     <x>444</x>
     <y>3</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>doubleSpinBox_Sy</sender>
   <signal>valueChanged(double)</signal>
   <receiver>ScaleAndBiasDlg</receiver>
   <slot>ValuesChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>242</x>
     <y>132</y>
    </hint>
    <hint type="destinationlabel">
     <x>492</x>
     <y>27</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>doubleSpinBox_Sz</sender>
   <signal>valueChanged(double)</signal>
   <receiver>ScaleAndBiasDlg</receiver>
   <slot>ValuesChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>392</x>
     <y>130</y>
    </hint>
    <hint type="destinationlabel">
     <x>514</x>
     <y>50</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>doubleSpinBox_Bx</sender>
   <signal>valueChanged(double)</signal>
   <receiver>ScaleAndBiasDlg</receiver>
   <slot>ValuesChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>92</x>
     <y>190</y>
    </hint>
    <hint type="destinationlabel">
     <x>512</x>
     <y>77</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>doubleSpinBox_By</sender>
   <signal>valueChanged(double)</signal>
   <receiver>ScaleAndBiasDlg</receiver>
   <slot>ValuesChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>253</x>
     <y>190</y>
    </hint>
    <hint type="destinationlabel">
     <x>515</x>
     <y>105</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>doubleSpinBox_Bz</sender>
   <signal>valueChanged(double)</signal>
   <receiver>ScaleAndBiasDlg</receiver>
   <slot>ValuesChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>430</x>
     <y>191</y>
    </hint>
    <hint type="destinationlabel">
     <x>511</x>
     <y>133</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>ScaleIsotropic()</slot>
  <slot>ScaleUnisotropic()</slot>
  <slot>ValuesChanged()</slot>
 </slots>
</ui>
